esp/esdllib/esdl_transformer2.cpp

```diff
@@ -1317,7 +1317,7 @@ void Esdl2Response::process(Esdl2TransformerContext &ctx, const char *out_name,
 
                     if (ctx.schemaLocation.length() > 0 )
                     {
-                        ctx.writer->outputXmlns("xmlns:xsi", "http://www.w3.org/2001/XMLSchema-instance");
+                        ctx.writer->outputXmlns("xsi", "http://www.w3.org/2001/XMLSchema-instance");
                         ctx.writer->outputXmlns("xsi:schemaLocation", ctx.schemaLocation.str());
```

Comment:
This is not a namespace, and shouldn't call outputXmlns.  This is an attribute that uses the "xsi"  namespace.